src/lfc_scripts.py

```diff
@@ -2,54 +2,79 @@
 import config
 from geopy.geocoders import Nominatim
 import folium
+from folium import FeatureGroup, LayerControl, Map, Marker
+import pandas as pd
+import sys
+
+site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+site.login(config.username, config.api_key)
+geolocator = Nominatim(user_agent = "map")
 
 def main():
-    COMPETITION_NAME = "LoneStar2020"
-    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-    site.login(config.username, config.api_key)
+    #Parse arguments
+    competition = sys.argv[1]
+    color = sys.argv[2]
+
+    #Setup Map
+    mapit = folium.Map(location=[48, -102], zoom_start=6)
+    loc = 'Corpus Christi'
+    title_html = '''
+                 <h3 align="center" style="font-size:16px"><b>Applicant Location Maps</b></h3>
+                 '''.format(loc)
 
+    #Retrieve proposal ids from competition
     response = site.api(
         'torquedataconnect',
         format='json',
-        path='/competitions/' + COMPETITION_NAME + '/proposals'
+        path='/competitions/' + competition + '/proposals'
     )
     proposal_ids = response["result"]
 
-    geolocator = Nominatim(user_agent = "map")
+    locations = extract_locations(proposal_ids, competition) #Extract locations from proposal ids
 
+    feature_group = FeatureGroup(competition)
+    for coord in locations:
+        folium.CircleMarker(location=[coord[0], coord[1]], fill_color=color, radius=7, color = color, opacity = 0.5, weight = 1).add_to(feature_group)
```

Comment:
This line is pretty long -- may be good to break the params into their own lines

e.g.

```
folium.CircleMarker(
  location = [...],
  fill_color = ...,
  ...
).add_to(feature_group)
```

Reply:
resolved